### ISSUE TITLE
[20220424] Weekly AI ArXiv 만담 - 49회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#49]
### News
- 컨퍼런스
  - ICLR 2022: 
    - 네이버 클로바 발표 스케쥴: https://naver-career.gitbook.io/en/teams/clova-cic/events/naver-clova-iclr-2022
    - ML in Korea: https://naver-career.gitbook.io/en/teams/clova-cic/events/iclr-2022-social-ml-in-korea
- [CLIP ViT-L/14@336px checkpoint 공개!](https://github.com/openai/CLIP/commit/b4ae44927b78d0093b556e3ce43cbdcff422017a)
- [LAION-5B: 최대규모 이미지-텍스트 공개 데이터](https://laion.ai/laion-5b-a-new-era-of-open-large-scale-multi-modal-datasets/)
 
### ArXiv
- [DiffCSE: Difference-based Contrastive Learning for Sentence Embeddings](https://arxiv.org/abs/2204.10298)
  - MIT, MIT-IBM 에서 나온 sentence embedding 연구 (NAACL 2022 메인)
  - Constrastive learning을 통해 Sentence embedding 학습
  - Dropout 기반 Transformer를 활용하는 [SimCSE](https://arxiv.org/abs/2104.08821) 에 마스크된 부분을 생성하고 생성된 문장과 원본문장 사이의 차이를 예측토록하는 difference-based CL도 함께 학습 
  - https://github.com/voidism/DiffCSE
![image](https://user-images.githubusercontent.com/11782739/164972553-6e284369-fbab-4f66-bbdd-45533a3ea8fa.png)

- [Do Prompt-Based Models Really Understand the Meaning of their Prompts?](https://arxiv.org/abs/2109.01247)
  - Brown Univ 에서 나온 Large LM에서 prompt가 잘되는 것이 instruction 효과인가에 대한 고찰 (NAACL 2022)
  - Prompt-based learning이 잘되는 것은 사람처럼 introduction에 의해 주어진 정보때문이라고 추정함.
  - 실제로 그러한지를 실험적으로 검증해봄
  - 결론적으로 말하면 instruction을 관련없거나.. 잘못해도 크게 상관없이 즉 prompt가 있기만 하면 few-shot 성능은 좋아지더라는...
  - 그래서.. prompt의 instruction을 이해하는 것이 아니었다....
  - https://openreview.net/forum?id=BhGMkxhZrW9: NAACL 2022 버전
![image](https://user-images.githubusercontent.com/11782739/164972227-064ee42c-a8e8-4d4b-9668-b86f0dcd6625.png)

- 흥미있는 연구
  - [StyleGAN-Human: A Data-Centric Odyssey of Human Generation](https://github.com/stylegan-human/StyleGAN-Human)
    - 얼굴이 아닌 전신생성!! 선코드 후논문 (논문은 5월전에 나온다고..)
  - [SmartPortraits: Depth Powered Handheld Smartphone Dataset of Human Portraits for State Estimation, Reconstruction and Synthesis](https://arxiv.org/abs/2204.10211)
    - 대량의 스마트폰 비디오 상반신 데이터셋. 
  - [Depth-Aware Generative Adversarial Network for Talking Head Video Generation](https://github.com/harlanhong/CVPR2022-DaGAN)
    - Talking head 연구하시는 분들은 참고하면 좋을 듯?

**Comment by ghlee0304** [comment]
### Arxiv
- [FastDiff: A Fast Conditional Diffusion Model for High-Quality Speech Synthesis](https://arxiv.org/abs/2204.09934)
- [A Post Auto-regressive GAN Vocoder Focused on Spectrum Fracture](https://arxiv.org/abs/2204.06086)
- 흥미있는 연구 
    - [BYOL for Audio: Exploring Pre-trained General-purpose Audio Representations](https://arxiv.org/abs/2204.07402)

**Comment by veritas9872** [comment]
Arxiv
[Planting Undetectable Backdoors in Machine Learning Models](https://arxiv.org/abs/2204.06974)

**Comment by kimyoungdo0122** [comment]
- News
    - 중국의 AI 연구기관 BAAI에서 논문을 표절했다?!
    - [Chinese Prominent AI Lab Plagiarizes Big Model Paper](https://recodechinaai.medium.com/chinese-prominent-ai-lab-plagiarizes-big-model-paper-microsoft-research-asia-halts-internship-ea645056c788)
    - 표절해서 작성한 논문은 예전에 weekly arxiv에서 소개드셨던 'A Roadmap for Big Model'이라는 논문 제목으로, 수조 개의 파라미터를 구현 가능하다고 주장하며 'Brain-scale AI'라고 표현했던 그 [연구](https://mixed-news.com/en/artificial-intelligence-china-researches-brain-scale-ai/) 
![image](https://user-images.githubusercontent.com/42539153/164972985-78187fd6-c60c-4f96-974b-ffcaa8a38dfd.png)
    - 현재 Arxiv에도 페이퍼가 내려와있는 상황 
    - 표절당한 논문은 구글 브레인의 [Deduplicating Training Data Makes Language Models Better](https://arxiv.org/abs/2107.06499)라는 연구의 일부분
    - 트위터에서도 구글 브레인의 연구자 Katharine Lee에 의해 알려졌고, Nicholas Carlini는 [개인 홈페이지](https://nicholas.carlini.com/writing/2022/a-case-of-plagarism-in-machine-learning.html)에 표절에 대한 근거와 의견 제시
    - BAAI 홈페이지에 게재된 사과문 (중국어는 도저히 못읽겠어서 번역돌렸습니다)
    - 한국의 자랑스런 AI 연구자들께서도 조심하세요..!!(표절 하시는 것도, 당하시는 것도!!)
![image](https://user-images.githubusercontent.com/42539153/164973383-e536bc72-de8a-4e74-9cb0-7e06be3d0868.png)
![image](https://user-images.githubusercontent.com/42539153/164973456-2b4a9512-6592-46c2-82a8-2cd903b262f8.png)

**Comment by nick-jhlee** [comment]
## News
- KCC 2022 

## Arxiv (ICLR preview)
### CV, NLP
- [Autoregressive Image Generation using Residual Quantization](https://arxiv.org/abs/2203.01941)
   - Kakao Brain, POSTECH CV Lab
   - Github link 공개: https://github.com/lucidrains/RQ-Transformer
   
- [Exploring the Limits of Large Scale Pre-training](https://openreview.net/forum?id=V3C8p78sDa)
   - Google Research
   - 

- [Fairness in Representation for Multilingual NLP: Insights from Controlled Experiments on Conditional Language Modeling]   - 
   - 단독 저자! (Ada Wan, University of Zurich)

(https://openreview.net/forum?id=-llS6TiOew)
### Theory
- [SGD with a Constant Large Learning Rate Can Converge to Local Maxima](https://openreview.net/forum?id=9XhPLAjjRB) (ICLR 2022 Spotlight)
   - University of Tokyo, ENS PSL, UC Berkeley
   - 말 그대로, SGD를 저흰 아직 완벽하게 이해하지 못함!
   - 물론, theoretical paper답게, toy landscape를 고려함 (non-convex loss + multi-minima)
   - Contributions:
       - SGD may converge to a local maximum
       - SGD may escape a saddle point arbitrarily slowly
       - SGD may prefer sharp minima over flat ones
       - AMSGrad may converge to a local maximum
   
![image](https://user-images.githubusercontent.com/38721329/164974559-5cd1fbe2-18a5-459d-bc8f-5c1af18cf1e3.png)
![image](https://user-images.githubusercontent.com/38721329/164974575-20712e50-d2af-4936-ac43-49b26f5da513.png)
